investor/LoanFilter.py

```diff
@@ -23,16 +24,11 @@ def __str__(self):
 		pass
 
 	@abstractmethod
-	def _eval(self, loan):
+	def _eval(self, loan, block):
 		pass
 
-	def apply(self, loan):
-		if self._eval(loan):
-			self.pass_count += 1
-			return True
-		else:
-			self.fail_count += 1
-			return False
+	def apply(self, loan, block=True):
+		return self._eval(loan, block)
```

Comment:
Alright, I misunderstood the problem. Didn't realize that we may not have the result at this point. Thinking more on it, I think we can remove the tallying functionality. It's trivial to implement elsewhere, and we haven't implemented anything that takes advantage of it yet.